lightning/src/chain/chaininterface.rs

```diff
@@ -19,8 +19,8 @@ use bitcoin::blockdata::transaction::Transaction;
 
 /// An interface to send a transaction to the Bitcoin network.
 pub trait BroadcasterInterface {
-	/// Sends a transaction out to (hopefully) be mined.
-	fn broadcast_transaction(&self, tx: &Transaction);
+	/// Sends a list of transactions out to (hopefully) be mined.
+	fn broadcast_transactions(&self, txs: &[&Transaction]);
```

Comment:
I think this is more probable, we’ll add a `broadcast_package_v3` call as package processing will come with new policy rules more constraining than the usual transaction broadcast. What’s the underlying transaction-related Core’s RPC call are you thinking to use here ? We have a bunch of them `sendrawtransaction`, `submitpackage`, `testmempoolaccept` and some are still experimental.

Reply:
Yeah didn't realize until after I made this that `submitpackage` is regtest only. Still think you could get some wins by broadcasting multiple transactions concurrently for now.

Reply:
> Core has to be able to figure out, given its current mempool state, how to submit packages to its peers

That’s the theory and this one I shared and an approach I advocated for in the past. In practice for the current design of nversion=3 there is a responsibility on the user to ensure package are sane (i.e `CheckPackage()`), or at least this is my understanding of the current API, which is one of the most active and worked-on area of Bitcoin Core. There is a https://github.com/bitcoin/bitcoin/issues/27463 if anyone has feedback as a second-layer dev.

Reply:
>What’s the underlying transaction-related Core’s RPC call are you thinking to use here ? 

Maybe it is a NIP-89 post?

Reply:
> My point above was that, once we get our transactions into the mempool, 

So I think this what my point said, to ensure our transactions get into the mempool they will have to respect nversion=3 format as requested by the RPC API’s check like `CheckPackage()` (in `AcceptPackage` and `AcceptMultipleTransactions`) and this is related to LDK. We’ll just have to support mempool rules checks implemented like we have already in our transaction processing backend about RBF policy (e.g in `feerate_bump`).